core/domain/classifier_services.py

```diff
@@ -305,9 +287,9 @@ def _update_classifier_training_job(classifier_training_job_model, status):
     classifier_training_job_model.put()
 
 
-def save_classifier_training_job(algorithm_id, exp_id, exp_version,
-                                 state_name, status, training_data,
-                                 job_id="None"):
+def save_classifier_training_job(algorithm_id, interaction_id, exp_id,
+                                 exp_version, state_name, training_data,
+                                 status, job_id=None):
```

Comment:
job_id still should not have a default arg. I think you can actually just pass in the domain object. See how the similar methods in exp_services do it, for example. But the domain object should be "complete" and contain all necessary fields.